modular_skyrat/modules/colony_fabricator/code/design_datums/flatpack_machines.dm

```diff
@@ -0,0 +1,171 @@
+// Machine categories
+
+#define FABRICATOR_CATEGORY_FLATPACK_MACHINES "/Flatpacked Machines"
+#define FABRICATOR_SUBCATEGORY_MANUFACTURING "/Manufacturing"
+#define FABRICATOR_SUBCATEGORY_POWER "/Power"
+#define FABRICATOR_SUBCATEGORY_MATERIALS "/Materials"
+
+// Techweb node that shouldnt show up anywhere ever specifically for the fabricator to work with
+
+/datum/techweb_node/colony_fabricator_flatpacks
+	id = "colony_fabricator_flatpacks"
+	display_name = "Colony Fabricator Flatpack Designs"
+	description = "Contains all of the colony fabricator's flatpack machine designs."
+	design_ids = list(
+		"flatpack_solar_panel",
+		"flatpack_solar_tracker",
+		"flatpack_arc_furnace",
+		"flatpack_colony_fab",
+		"flatpack_station_battery",
+		"flatpack_station_battery_large",
+		"flatpack_fuel_generator",
+		"flatpack_rtg",
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000000000000000) // God save you
+	hidden = TRUE
+	show_on_wiki = FALSE
+	starting_node = TRUE
+
+// Lets the colony lathe make more colony lathes but at very hihg cost, for fun
```

Comment:
```suggestion
// Lets the colony lathe make more colony lathes but at very high cost, for fun
```
OK so the earlier comment I pointed out WAS meant to be relating to the multicell

Reply:
soul

Reply:
solu

Reply:
so true